src/store/d-apps/d-apps-epics.ts

```diff
@@ -142,6 +151,36 @@ const approveOperationRequestEpic = (action$: Observable<Action>) =>
     )
   );
 
+const waitForOperationCompletionEpic = (action$: Observable<Action>) =>
+  action$.pipe(
+    ofType(waitForOperationCompletionAction),
+    toPayload(),
+    withLatestFrom(tezos$),
+    switchMap(([{ opHash, sender }, tezos]) =>
+      from(tezos.operation.createOperation(opHash)).pipe(
+        switchMap(operation => from(operation.confirmation(1))),
+        switchMap(({ completed }) =>
+          completed
+            ? of(null).pipe(
+                delay(15000),
```

Comment:
please, create variable for this magic number (`DATA_UPDATE_DELAY` for ex.)

Reply:
And let's try with smaller delay - 10sec

Reply:
I tried exactly 10s it a long time ago but there was a case when it wasn't enough